arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1586,12 +1699,12 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition for other than namespace."
     },
-    "NamespaceListResult": {
+    "SBNamespaceListResult": {
```

Comment:
Yes however we've already shipped SDKs that contain types with these names, so if you change them now you'll be introducing breaking changes.  I believe these changes will need to be reverted, @dsgouda @amarzavery do you agree?